docs/source/users/index.md

```diff
@@ -75,7 +75,7 @@ Before you send your first prompt to an AI model, load the IPython extension.
 Enter the following code into a Jupyter Notebook cell and run the cell:
 
 ```
-%load_ext jupyter_ai
+%load_ext jupyter_ai_magics
```

Comment:
We should not remove `%load_ext jupyter_ai` from the docs and example NB; we should merely state that users may install `jupyter_ai_magics` and use `%load_ext jupyter_ai_magics` if you don't want the labextension.